docs/performance_tuning.md

```diff
@@ -78,15 +78,15 @@ OpenVINO&trade; Model Server can be tuned to a single client use case or a high
 execution streams. They split the available resources to perform parallel execution of multiple requests.
 It is particularly efficient for models which cannot effectively consume all CPU cores or for CPUs with high number of cores.
 
-By default, number of streams is calculated based on number of available CPUs. It gives a compromise between the single client scenario and the high concurrency.
+By default, number of streams is optimized for execution with minimal latency with low concurency. 
 If this default configuration is not suitable, adjust it with the `NUM_STREAMS` parameter defined as part 
-of the device plugin configuration. 
+of the device plugin configuration or set the performance hint to `throughput`. 
 
-In a scenario where the number of parallel connections is close to 1, set the following parameter:
+In a scenario with a single connections is 1, set the following parameter:
 
-`--plugin_config '{"NUM_STREAMS": "1"}'`
+`--plugin_config '{"PERFORMANCE_HINT": "THROUGHPUT"}'`
 
-When the number of concurrent requests is higher, increase the number of streams. Make sure, however, that the number of streams is lower than the average volume of concurrent inference operations. Otherwise, the server might not be fully utilized.
+When the number of concurrent requests is high, adjust the number of streams. Make sure, however, that the number of streams is lower than the average volum of concurrent inference operations. Otherwise, the server might not be fully utilized.
```

Comment:
```suggestion
When the number of concurrent requests is high, adjust the number of streams. Make sure, however, that the number of streams is lower than the average volume of concurrent inference operations. Otherwise, the server might not be fully utilized.
```